docs/source/lifted-transformers/lifted-transformers.rst

```diff
@@ -1,7 +1,7 @@
 .. _lifted-transformers:
 
-Lifted transformers
-===================
+Lifted Transformers ``(deprecated)``
+====================================
 
 .. warning::
```

Comment:
Same as unsafeOption - we might explain in the warning that this is scheduled for deletion in probably 0.8.0

Reply:
Same as above, no promises as general rule.